docs/src/format.md

```diff
@@ -0,0 +1,400 @@
+| net1\_layer1_ps     | lin                | -inf           | inf            | 1          | netId1\_ps     | kaimingUniform              | 1                                   |
+| net1\_layer2_ps     | lin                | -inf           | inf            | 1          | netId1\_ps     | kaimingUniform              | 5/3                                 |
+
+Where `parameterId` are assumed to have been properly assigned in the [mapping table](@ref mapping_table). In this example, each layer references the same file variable for `nominalValue`. This means the layers obtain their values from the specified file.
+
+It is also possible to specify different priors for different parameter arrays within a layer. For example, to use different priors for the weights and bias in `layer1` of `net1` (assuming a `linear` layer), a valid parameter table would be:
+
+| **parameterId**      | **parameterScale** | **lowerBound** | **upperBound** | **estimate** | **nominalValue** | **initializationDistribution** | **initializationDistributionParameters** |
+|----------------------|--------------------|----------------|----------------|------------|------------------|-----------------------------|-------------------------------------|
+| net1\_ps              | lin                | -inf           | inf            | 1          | netId1\_ps     | kaimingUniform              | 1                                   |
+| net1\_layer1_weight   | lin                | -inf           | inf            | 1          | netId1\_ps     | kaimingUniform              | 1                                   |
+| net1\_layer1_bias     | lin                | -inf           | inf            | 1          | netId1\_ps     | kaimingNormal               | 5/3                                 |
+
+## [Problem YAML File](@id YAML_file)
+
+The PEtab problem YAML file follows the PEtab version 2 format, except that a mapping table is now required (it is optional in the standard). It also includes an extension section for specifying neural network YAML files, as well as any files for neural network parameters and/or inputs. Specifically the extension sections includes two new groups:
+
+- `extensions > petab_sciml > neural_nets`: Here each neural network is defined via a key-value mapping, where the key is the neural network Id. The corresponding value is an object whose fields depend on the network format:
+  - For models in the `petab_sciml` format, include the file path as `location`.
+  - For neural networks in other formats, includes the neural network library name as `library`.
+- `extensions > petab_sciml > array_files`: Here each associated neural network array file. Parameter files should follow the structure [here](@ref hdf5_ps_structure), and input files the structure [here](@ref hdf5_input_structure). Each entry is specified with a key-value mapping, where the key is the array Id which is considered a valid PEtab Id. The corresponding value is an object that includes:

```

Comment:
```suggestion
- `extensions > petab_sciml > array_files`: Specifies array files. Parameter files should follow the structure [here](@ref hdf5_ps_structure), and input files the structure [here](@ref hdf5_input_structure). Each entry is specified with a key-value mapping, where the key is the array Id which is considered a valid PEtab Id. The corresponding value is a dict with:

```

Don't add links with body `here`, it's just fluff, directly link the thing you are referring to.